maskinporten_api/permissions.py

```diff
@@ -37,8 +22,28 @@ def create_okdata_permissions(
 
 
 def get_user_permissions(bearer_token):
-    return requests.get(
+    res = requests.get(
         f"{OKDATA_PERMISSION_API_URL}/my_permissions",
         headers={"Authorization": f"Bearer {bearer_token}"},
         params={"resource_type": "maskinporten:client"},
-    ).json()
+    )
+    res.raise_for_status()
+    return res.json()
+
+
+def get_user_team(team_id, bearer_token, has_role=None):
+    params = {}
+
+    if has_role:
+        params["has_role"] = has_role
+
+    res = requests.get(
+        f"{OKDATA_PERMISSION_API_URL}/teams",
+        headers={"Authorization": f"Bearer {bearer_token}"},
+        params=params,
+    )
+    res.raise_for_status()
+    for team in res.json():
+        if team["id"] == team_id:
+            return team
+    return None
```

Comment:
Bruke den nye utgivelsen av okdata-sdk for dette?

Reply:
Kanskje den kan rebases på https://github.com/oslokommune/okdata-maskinporten-api/pull/67.

Reply:
^ Never mind, ref. diskusjon.